docs/conf.py

```diff
@@ -229,13 +249,12 @@
     "meth": "tooltip",
     "mod": "tooltip",
     "obj": "tooltip",
-    # roles with std domain
+    # Roles within the standard domain
```

Comment:
the std domain is a *thing*

```suggestion
    # Roles within the std domain
```

Reply:
I will accept after RTD builds.